src/ClusterBootstrap/cloud_init_deploy.py

```diff
@@ -50,21 +50,40 @@ def get_platform_script_directory(target):
 
 default_config_mapping = {
     "dockerprefix": (["cluster_name"], lambda x: x.lower()),
-    "infrastructure-dockerregistry": (["dockerregistry"], lambda x: x),  # keep
-    "worker-dockerregistry": (["dockerregistry"], lambda x: x),  # keep
-    "api-server-ip": (["service_cluster_ip_range"], lambda x: generate_ip_from_cluster(x, 1)),
-    "dns-server-ip": (["service_cluster_ip_range"], lambda x: generate_ip_from_cluster(x, 53)),
-    "network-trusted-domains": (["network"], lambda x: generate_trusted_domains(x, 5)),
+    "infrastructure-dockerregistry": (["dockerregistry"], lambda x: x), # keep
```

Comment:
There could be conflicts between this PR and #1173. +@YinYangOfDao for awareness.

Reply:
I'll reset and disable format and edit again.